packages/gateway/src/features/registration/fhir-builders.ts

```diff
@@ -3635,9 +3637,40 @@ export async function buildFHIRBundle(
     builders,
     context
   )
+  const isADuplicate =
+    eventType === EVENT_TYPE.BIRTH &&
+    (await hasBirthDuplicates(authHeader, reg as GQLBirthRegistrationInput))
+
+  if (isADuplicate) {
+    composition.resource.extension = composition.resource.extension || []
+    composition.resource.extension.push({
+      url: `${OPENCRVS_SPECIFICATION_URL}duplicate`,
+      valueBoolean: true
+    })
+  }
   return fhirBundle
 }
 
+async function hasBirthDuplicates(
+  authHeader: IAuthHeader,
+  bundle: GQLBirthRegistrationInput
+) {
+  if (!bundle || !bundle.child) {
+    return false
+  }
+
+  const res = await findDuplicates(authHeader, {
```

Comment:
This causes a lot of tests to need to be rewritten; as many of them are mocked response by response, and this creates a new request/response in some cases.